Dockerfile.options

```
@@ -31,7 +31,7 @@ ENV MAKE_INCLUDES="Makefile Makefile.*"
 # This is separate so that updating it does not invalidate the Docker cache layer with all the packages installed above
 # https://cloud.google.com/sdk/docs/release-notes
 ARG GOOGLE_CLOUD_CLI_VERSION
-ENV CLOUDSDK_CONFIG=/localhost/.config/gcloud/
+ENV CLOUDSDK_CONFIG=/etc/xdg_config_home/.config/gcloud/
```

Hadolint (2.12.0)

[error] Invalid instruction order. Dockerfile must begin with `FROM`, `ARG` or comment.

(DL3061)